lib/call.js

```diff
@@ -3446,6 +3446,12 @@ class call {
       newcall.#onnewuaccatch( err )
     }
 
+    // Might have been picked by the queue
+    if ( newcall.state.picked ) {
+      newcall.hangup()
```

Comment:
@tinpotnick I am not really sure hangup is a good way to do it here, but I don't seem to be able to exclude either of the cases since drachtio is handling it?

Reply:
I don't understand what this would acheive. Any call which is picked (before an establish) would immediately be hung up?